client/question/domain/FeedbackParagraphObjectFactory.js

```diff
@@ -56,6 +56,14 @@ tie.factory('FeedbackParagraphObjectFactory', [
        * @private
        */
       this._content = content;
+
+      /**
+       * Line number that correlates to error that triggered.
+       *
+       * @type {integer}
+       * @private
+       */
+      this._errorLineNumber = (errorLineNumber) ? errorLineNumber : null;
```

Comment:
`this._errorLineNumber = errorLineNumber | null;` is more JavaScript-y. If the first operand is falsy then the evaluated value is the second operand.

Could you also add a note in the docstring about whether this is 1- or 0-indexed?